### DISCUSSION TITLE
Folder view

### DISCUSSION THREAD

**Discussion by davidbozo** [boyter/scc#268] [Q&A] [answered]
Hi!

I'm looking for a tool which can display line of codes by folder for a while. I tried to find it in the doc but i couldn't. 
Is there any flag like this?

**Comment by boyter** [comment, marked as answer]
You mean broken down by folder? As in you want to see how many lines are in say the directory `src` compared to `assets` for example?

If I am correct on this not out of the box person. You could however use the SQL format output to compute this however https://github.com/boyter/scc#sql-and-sql-insert and I believe that the cloc examples (linked there) include something like this.

**Reply by davidbozo** [reply]
I didn't find the tree example but this is a good starting point for me. Thank you!

**Reply by boyter** [reply]
Cool. If you find something neat be sure to let me know here and ill add it to the readme.
